clients/python-apiserver-client/python_apiserver_client/params/cluster.py

```diff
@@ -0,0 +1,193 @@
+from .headnode import *
+from .workernode import *
+
+class Environment(enum.Enum):
```

Comment:
Is this too specific for general use? What if I'm in a big environment with different kinds of clusters in each of these categories?